protocol/x/clob/memclob/memclob.go

```diff
@@ -1004,7 +1004,7 @@ func (m *MemClobPriceTimePriority) ReplayOperations(
 			}
 
 			// Log an error if there are two Order Removals for the same OrderId
-			if _, found := placedPreexistingStatefulOrderIds[orderId]; found {
+			if _, found := placedOrderRemovalOrderIds[orderId]; found {
```

Comment:
This check was previously the exact same as the above check, which was an oversight